framework/set/provisioning/nodedriver/rke2k3s/setConfig.go

```diff
@@ -111,6 +112,16 @@ func SetRKE2K3s(client *rancher.Client, terraformConfig *config.TerraformConfig,
 
 	rkeConfigBlock := clusterBlockBody.AppendNewBlock(defaults.RkeConfig, nil)
 	rkeConfigBlockBody := rkeConfigBlock.Body()
+	// CNI Chart Block
+	chartValues := hclwrite.TokensForTraversal(hcl.Traversal{
+		hcl.TraverseRoot{Name: "<<EOF\n" + terraformConfig.ChartValues + "\nEOF"},
+	})
+	rkeConfigBlockBody.SetAttributeRaw(defaults.ChartValues, chartValues)
```

Comment:
Question:
If terraformConfig.ChartValues is `nil`, does this break anything, having it declared in the `main.tf` without a value assigned? 🤔 -- If so, could you wrap this in validation to ensure its only declared if ChartValues are present in the config? Something like `if terraformConfig.ChartValues != nil {}`

Reply:
Agreed. The etcd block is an example on how to do this. As of now, this is going to mandate this be defined in the config, which will break other usecases.